include/blast/math/expressions/PanelMatrix.hpp

```diff
@@ -185,22 +187,22 @@ namespace blast
 
         using ET1 = ElementType_t<MT1>;
         using ET2 = ElementType_t<MT2>;
-        static size_t constexpr SS = Simd<ET2>::size;
+        static size_t constexpr SS = SimdSize_v<ET2>;
         static size_t constexpr PANEL_SIZE = PanelSize_v<ET2>;
         static_assert(PANEL_SIZE % SS == 0);
 
-        using MaskType = typename Simd<ET2>::MaskType;
-        using IntType = typename Simd<ET2>::IntType;
+        using MaskType = SimdMask<ET2, xsimd::default_arch>;
```

Comment:
The `xsimd::default_arch` is implied here I guess? Not really necessary?

Reply:
True. I think I wrote this line before I added a default value for `Arch` in `SimdMask`.